second-gen/packages/core/components/badge/Badge.base.ts

```diff
@@ -61,26 +69,14 @@ export abstract class BadgeBase extends SizedMixin(
     @property({ type: String, reflect: true })
     public variant: BadgeVariant = 'informative';
 
-    @property({ reflect: true })
-    public get fixed(): FixedValues | undefined {
-        return this._fixed;
-    }
+    @property({ type: Boolean, reflect: true })
+    public subtle: boolean = false;
 
-    public set fixed(fixed: FixedValues | undefined) {
```

Comment:
This seems to me to be how the @property works by default. I don't see any regressions when I simplify this but let me know if I've missed a nuance here.